### PR TITLE
Arc-2413 auto consent

### PR DESCRIPTION
**Pull request by rachellerathbone**
**What's in this PR?**
Added autoUserConsent to manifest file

**Why**
So when users install the app, they don't need accept permissions

Screenshot 2023-08-16 at 4 28 40 pm

**Affected issues**  
Arc-2413

**How has this been tested?**  
Locally

### CODE REVIEW

File: app/jenkins-for-jira-ui/.env.local
Comment (krazziekay) [inline app/jenkins-for-jira-ui/.env.local:1]: Please remove this file as it contains your dev env variables.

**Comment by yvonnevictoria** [comment]
So great we've managed to fix this issue!

File: app/jenkins-for-jira-ui/.env.local
Comment (rachellerathbone) [reply app/jenkins-for-jira-ui/.env.local:1]: wtf, I removed this and removed it from the cache :blob-confused:

**Review by krazziekay — APPROVED** [review]

**Review by yvonnevictoria — APPROVED** [review]

File: app/jenkins-for-jira-ui/.env.local
Comment (yvonnevictoria) [reply app/jenkins-for-jira-ui/.env.local:1]: What does your `.gitignore` look like? @rachellerathbone

**Pull request merged**